lib/fluent/plugin/in_wmi.rb

```diff
@@ -13,14 +14,29 @@ def configure(conf)
 
     def start
       super
-      tag = "wmi.test"
+      wmi = WmiLite::Wmi.new
+      instances = wmi.instances_of('Win32_Processor')
+      tag = "monitor.wmi"

```

Comment:
I notice we have a great opporunity here to introduce the configuration
parameter support to `fluent-plugin-in-wmi`.

It should be as easy as just a few tweaks to the code:

 1. Define a pair of `config_param` and `desc` onto this plugin (see the
    following example for how to do it)

    https://github.com/fluent/fluentd/blob/master/lib/fluent/plugin/in_dummy.rb#L31-L32

 2. Modify the code to use the variable `@tag`, instead of the predefined
     string literal.

... and it will enable us to change the tag value as we like through
configuration file.

Reply:
Thanks for your good suggestion. How to define configuration was my next research things. I will try it.